### DISCUSSION TITLE
Join us and take TopoToolbox to the next level

### DISCUSSION THREAD

**Discussion by wschwanghart** [wschwanghart/topotoolbox#41] [Announcements]
_see also [here](https://topotoolbox.wordpress.com/2023/11/03/join-us-and-take-topotoolbox-to-the-next-level/)_

TopoToolbox has now been around for more than a decade. Yet, there are many new developments needed to take TopoToolbox to the next level. This requires a lot of work, and thus we are happy to announce that the [German Research Foundation (DFG)](https://www.dfg.de/en/index.jsp) granted us a project that enables us to hire a PostDoc/Research Software Engineer to support our team.

[Please check the official job announcement here](https://topotoolbox.files.wordpress.com/2023/11/421_2023_topotoolbox_dfg_en.pdf) ([german version](https://topotoolbox.files.wordpress.com/2023/11/421_2023_topotoolbox_dfg_de.pdf)).

We are seeking a skilled and motivated PostDoc/Research Software Engineer to join our team for the DFG-funded project “TopoToolbox 3 – improving the quality and reuse of a research software for terrain analysis”. Our team, that’s [Wolfgang Schwanghart](https://www.uni-potsdam.de/de/umwelt/institut/alle-mitarbeiterinnen/schwanghart-wolfgang), [Dirk Scherler](https://sites.google.com/site/scherlerdirk/home) and [Anna-Lena Lamprecht](https://www.uni-potsdam.de/de/cs-se/team/prof-dr-anna-lena-lamprecht). You will contribute to the further development of TopoToolbox, a versatile and widely used open-source toolbox for digital elevation model (DEM) analysis. The aims of the project are i) to improve the usability and accessibility of TopoToolbox by a broad academic research community, ii) to improve the quality assurance in the software’s development process, and iii) to increase the involvement of the research community in the ongoing development of TopoToolbox.

As a Research Software Engineer on the project, you will play a crucial role in expanding TopoToolbox’s functionality, enhancing its sustainability and usability, and ensuring it remains a valuable resource for the geoscientific and environmental research community. You will work closely with an engaged team of researchers from the geosciences and computational sciences, utilizing your expertise in high-level programming languages such as Python, MATLAB, R, and low-level languages like C or C++. This position offers an excellent opportunity to kickstart a career as a Research Software Engineer in a geoscientific and environmental setting. This position is fixed-term for the duration of the project (24 months) with the potential for extension based on performance and funding availability.

Please find additional information (requirements, benefits, how to apply) [here](https://topotoolbox.files.wordpress.com/2023/11/421_2023_topotoolbox_dfg_en.pdf). We are looking forward to receiving your application! If you have questions, don’t hesitate to [contact me](https://www.uni-potsdam.de/de/umwelt/institut/alle-mitarbeiterinnen/schwanghart-wolfgang).

**Comment by mohseniaref** [comment]
Is your python project  is now open in github?

**Reply by wschwanghart** [reply]
Hi Mohammad, wait for it ;-)
